src/app/login/login.component.html

```diff
@@ -11,7 +11,7 @@ <h1>Login</h1>
       <input matInput placeholder="Password" type="password" name="password" formControlName="password" required>
       <mat-icon matSuffix>lock</mat-icon>
     </mat-form-field>
-    <button mat-raised-button color="warn" type="submit">Login</button>
+    <button mat-raised-button color="primary" type="submit">Login</button>
```

Comment:
why?

Reply:
warn is in RED color for warning alert. We should keep Login as primary, it matches with the current theme. It's better.